README.md

```diff
@@ -1 +1,87 @@
-solution-pack-data-leakage-threat-response
\ No newline at end of file
+# Data Leakage Threat Response Solution Pack
+
+## Release Information
+
+- Solution Pack Version: 1.0.0
+- Minimum Compatible FortiSOAR™ Version: 7.2.0
+- Authored By: Fortinet
+- Certified: No
+
+## Overview
+
+### Introduction
+
+*Data Leakage Threat Response Solution Pack* is designed to provide a set of investigation and utility playbooks to respond to data leakage.
+
+Configure Cloud Access Security Broker(CASB) connector such as Symantec CloudSOC to capture events of policy violation like 'External File Sharing'. It creates an alert of type 'Data Leakage', and then triggers a response workflow.
+
+Refer to Simulation Scenarios - Data Leakage to experience the use case without CASB configuration
+
+This article describes the “Data Leakage Threat Response” Solutions Pack. The solution pack demonstrates the scenarios and use cases around Data Leakage.
+The below scenario demonstrates and generates a demo alert for the Alert.
+
+- An alert is generated once detected a user shares a file externally, which is a violation of policy
+- This triggers an investigation that determines if the employee is on the watchlist and if the file being shared is a sensitive document
+- In case a file is found to be a sensitive document then deactivate the user in AD and revoke file permission
```

Comment:
AD --> Microsoft Active Directory

Reply:
IMO we do not need this info as the same information is also given under the Usage section. Removing it altogether. Please review